alephium/contracts/sequence/sequence.ral

```diff
@@ -0,0 +1,62 @@
+// sequence started from 0, set `next = 0` when init contract state
+Abstract Contract Sequence(
+    mut next: U256,
+    mut next1: U256, // TODO: better name
+    mut next2: U256,
```

Comment:
Question: Instead of using `next1` and `next2` here, if we use one `sequence` field just like in the `UndoneSequence` contract, is the benefit that we will create less `UndoneSequence` sub contract?

Reply:
Yes, there is another benefit, the transaction execution order in the block may be inconsistent with the sequence order, if there is only one field in the `Sequence` contract, it may create an unnecessary `UndoneSequence` contract, and using two fields can greatly reduce the probability of this problem.